packages/@adobe/spectrum-css-temp/components/overlay/index.css

```diff
@@ -26,7 +26,7 @@ governing permissions and limitations under the License.
   visibility: visible;
   /* In Edge (pre chromium), a stacking context is formed for opacity less then 1, and then its removed for 1. 
      It causes a rendering flicker that is visible when css transition is applied. */
-  opacity: 0.99;
+  opacity: 0.9999;
```

Comment:
@pst67662 could you test this for me in Edge? Just wanna make sure that this still stops the stops the shrinking you noticed here: https://github.com/adobe-private/react-spectrum-v3/pull/410

Reply:
tested, works good!